src/polysolve/linear/Solver.hpp

```diff
@@ -4,12 +4,21 @@
 
 #include <memory>
 
+#include <Eigen/Dense>
+#include <Eigen/Sparse>
+#include <vector>
+
 #define POLYSOLVE_DELETE_MOVE_COPY(Base) \
     Base(Base &&) = delete;              \
     Base &operator=(Base &&) = delete;   \
     Base(const Base &) = delete;         \
     Base &operator=(const Base &) = delete;
 
+extern Eigen::MatrixXd init_vertices;
+extern Eigen::MatrixXd test_vertices;
+extern std::vector<int> test_boundary_nodes;
+Eigen::MatrixXd remove_boundary_vertices(const Eigen::MatrixXd &vertices, const std::vector<int> &boundary_nodes);
```

Comment:
this should be inside trilinos, not here

Reply:
Fixed